docs/src/index.md

```diff
@@ -34,17 +38,17 @@ julia> filter_df = df[ ( df.nequ .≥ 50000 ) .& ( df.nvar .≤ 34000 ), :]
    2 │ problem-73-11032-pre  ladybug  92244  33753  1106928
 ```
 
-`get_first_name_and_group` returns a tuple of the name and group in the first row of the dataframe
+You can get the problem name directly from the dataframe:
 
 ```julia
-julia> name, group = get_first_name_and_group(filter_df)
-("problem-49-7776-pre", "ladybug")
+julia> name = filter_df[1, :name]
+"problem-49-7776-pre"
 ```
```

Comment:
If you want to avoid this, you can use
```@example ex1
name = filter_df[1, :name]
```
This will execute the code and automatically print the result.

If for some reason, you need several environments, you can use a different tag, see `ex1` above.

Reply:
If I understand well I simply need to remove "julia >" in my code ?
In the case [here](https://github.com/JuliaSmoothOptimizers/BundleAdjustmentModels.jl/blob/13d757911befb8da3c738b7a57d89df73dcd6d92/docs/src/index.md?plain=1#L110) where the output is normally ugly what should I do ?

Reply:
Yes, you replace:
- julia by @example [tag]
- julia> are removed
The result of the last line of the block is printed. So, if it really doesn't print well, maybe you can add another line after.
For instance,
```
 jac_structure_residual!(model, rows, cols)
(length(rows), length(cols))
```